css/lodesiempre.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Bitter:wght@700&display=swap" rel="stylesheet">
+  <link href="styles.css" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Bitter:wght@700&family=Nunito&display=swap" rel="stylesheet">
+</head>
+
+<body>
+
+<div class="column-container">
+  <img src="onboarding-assets/img/titulo.png" style="width:502px; margin-top: 15px;">
+</div>
+
+<div class="Rectangle-2">
```

Comment:
the name Rectangle-2 looks weird to me. with is the 2 for? is this a class for the background? maybe .header-background is a better name for this (or something like that)